aireview/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG PREFECT_VERSION=3.1.5
+
+FROM prefecthq/prefect:${PREFECT_VERSION}-python3.11-kubernetes
+ARG POETRY_VERSION=1.8.3
+LABEL io.prefect.version=${PREFECT_VERSION}
+
+RUN pip install --no-cache-dir poetry==${POETRY_VERSION}
+
+WORKDIR /opt/prefect/flows
+COPY pyproject.toml poetry.lock ./
+
```

Comment:
**documentation**: Add .dockerignore file to exclude unnecessary files and potential secrets from being copied into the image